src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Random random = new Random();
+        FigureSupplier figureSupplier = new FigureSupplier();
+        FigureMain[] newFigure = new FigureMain[random.nextInt(50)];
```

Comment:
50 is a magic number - move it to a constant field
```suggestion
        FigureMain[] figures = new FigureMain[random.nextInt(50)];
```